src/site/content/en/blog/web-animations/index.md

```diff
@@ -0,0 +1,199 @@
+---
+title: The Web Animations API just hit Chrome stable — here's what you need to know
```

Comment:
Please make your title more like the [our other ones](https://web.dev/blog/).

Always avoid the terms "stable", "beta", and "canary". Articles will be on the web for years. These terms are only accurate for six weeks.